### DISCUSSION TITLE
How do I change the default options for a particular server?

### DISCUSSION THREAD

**Discussion by Juraci** [williamboman/nvim-lsp-installer#546] [Q&A] [answered]
Sorry for the really noob quesiton, I would like to change one of solagraph's default options `solargraph.useBundler` to `true` but I can't seem to find a way of doing it (I'm using lunarVim).
![screenshot_20220315_122757](https://user-images.githubusercontent.com/2266929/158412765-1f9c98c4-1d11-40d8-94b2-db64fd58b5b8.png)

Is there a way of interacting with these options? If not where is the config file that defines them?

**Comment by williamboman** [comment, marked as answer]
Hello! So that particular part of the UI is a read-only view of the server's configuration schema, and so it's entirely static. For changing the configurations of a server, there's an example snippet [here in the Wiki](https://github.com/williamboman/nvim-lsp-installer/wiki/Advanced-Configuration#overriding-the-default-lsp-server-options)

edit: This was recently clarified in the UI as well. As for better Solargraph/Ruby interop, see #187
